main.go

```diff
@@ -15,6 +15,7 @@ type Config struct {
 	AppSlug      string          `env:"BITRISE_APP_SLUG,required"`
 	AccessToken  stepconf.Secret `env:"access_token,required"`
 	BuildSlugs   string          `env:"buildslugs,required"`
+	SavePath     string          `env:"save_path"`
```

Comment:
in bitrise.yml context only `save_path` will be visible, which is not very descriptive.

wdyt about `build_artifacts_path` or any combination of these?